.github/scripts/windows/build_task.bat

```diff
@@ -32,7 +32,7 @@ if "%THREAD_SAFE%" equ "0" set ADD_CONF=%ADD_CONF% --disable-zts
 if "%INTRINSICS%" neq "" set ADD_CONF=%ADD_CONF% --enable-native-intrinsics=%INTRINSICS%
 if "%ASAN%" equ "1" set ADD_CONF=%ADD_CONF% --enable-sanitizer --enable-debug-pack
 
-set CFLAGS=/W2 /WX /w14013 /wd4146 /wd4244
+set CFLAGS=/W3 /WX /wd4018 /wd4146 /wd4244 /wd4267
```

Comment:
short comments above what these "numbers" are would greatly improve the readability

Reply:
Done. But I'm not 100% sure that makes much sense. One still needs to know what `/W3` implies, and as such might need to look up the warnings "all the time". In my opinion, gcc and clang handle this much better.